fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/services/ContainerServiceImpl.java

```diff
@@ -61,10 +63,22 @@ public Container findOrCreate(final Session session, final String path) {
 
             if (node.isNew()) {
                 initializeNewObjectProperties(node);
+
+                final Node parent = node.getDepth() > 0 ? node.getParent() : null;
+
+                if (parent != null) {
+                    touch(parent);
+                    touchLdpMembershipResource(node);
+                }
+            }
+
+            if (node.isNew()) {
+                touch(node);
             }
 
             return new ContainerImpl(node);
         } catch (final RepositoryException e) {
+            LOGGER.info("Error: {}", e);
```

Comment:
That also sounds fine to me.  The only issue I see with that approach is potentially wanting to have multiple logging statements to handle WARN/ERROR messages intended for integrators, and also DEBUG messages intended for programmers for the same exception, which seems cumbersome.  Does SLF4J or Logback provide some mechanism for configuring whether exceptions are logged as messages or stacktraces?